net/http/khttpmethod.c

```diff
@@ -37,4 +37,5 @@ const char kHttpMethod[18][8] = {
     "PATCH",    //
     "REPORT",   //
     "UNLOCK",   //
+    "LIST",     //
```

Comment:
Please update these lines to have 19 instead of 18.

```
http.h:190:extern const char kHttpMethod[18][8];
khttpmethod.c:21:const char kHttpMethod[18][8] = {
```

Reply:
Is there a reason to hardcode the methods at all? It might be easier to just remove the whole list and then any custom method will just work?

Reply:
As you wish skeleton jack. I'll push a change shortly that should allow you to use any method, with two caveats. Maximum method length is 8 chars. Upper casing is canonicalized. Any legal HTTP token is allowed.

Reply:
What's great is that doing methods the generalized way actually shaves 10 nanoseconds off the HTTP parser. I'm glad you challenged me on this issue. Redbean will be better as a result.

Reply:
I'll get myself a T-shirt that reads 'I helped shave 10 nanoseconds off Redbean's HTTP parser'